frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -234,7 +234,15 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'Page/Route',
     properties: {
       exact: true,
-      path: ['/add', '/import', '/topology', '/deploy-image', '/metrics', '/project-access'],
+      path: [
+        '/add',
+        '/import',
+        '/topology',
+        '/topology1',
```

Comment:
How long is this going to be around - can't we go with a better name if it has any shelf life?

Reply:
Removed in 4.3 after we use it for some testing.